### ISSUE TITLE
pretrain.model can not open, an error

### ISSUE THREAD

**Issue by zkailinzhang** [louis-she/center-loss.pytorch#1]
![_20180704151043](https://user-images.githubusercontent.com/10251153/42261998-794cadea-7f9c-11e8-8e8d-a8c0fb2d0833.png)

**Comment by zkailinzhang** [comment]
epoch_100.pth.tar  is really a true model ?,,not necessary  untar it ?

**Comment by hzh8311** [comment]
yes, you need not to untar it, load it with pytorch
